onyx-database-tests/src/streams/BasicQueryMapStream.java

```diff
@@ -0,0 +1,35 @@
+package streams;
+
+import com.onyx.exception.EntityException;
+import com.onyx.persistence.manager.PersistenceManager;
+import com.onyx.stream.QueryMapStream;
+import entities.identifiers.ImmutableSequenceIdentifierEntity;
+import entities.identifiers.ImmutableSequenceIdentifierEntityForDelete;
+
+import java.util.Map;
+
+/**
+ * Created by tosborn1 on 7/14/16.
+ */
+public class BasicQueryMapStream implements QueryMapStream{
```

Comment:
I thought we agreed on this convention a long while back that the interfaces should not be prefixed with I.  Such as PersistenceManager is not IPersistenceManager and the default implementation should be DefaultPersistenceManager.  

The same applies here.